apps/web/lib/error.ts

```
@@ -0,0 +1,17 @@
+export class AppError extends Error {
+    public statusCode: number;
+    public details?: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)